src/hooks/useFetch.js

```diff
@@ -105,10 +109,13 @@ export default () => {
 
       setData(responseData);
       setSuccess(true);
-    } catch (e) {
-      e.message = getErrorMessage(e);
+    } catch (exception) {
+      Object.defineProperty(exception, 'message', {
```

Comment:
Why is this construction needed here? Is it related to the translations?

Reply:
Not related; added it because otherwise the tests would break. Requires #775 to be merged first